pom.xml

```diff
@@ -48,6 +48,19 @@
     </developer>
   </developers>
 
+  <contributors>
+        <contributor>
+            <name>Manasse Ngudia</name>
+            <roles>
+                <role>developper contributor</role>
```

Comment:
Change this to "developer" (one letter 'p') please. Don't include word "contributor"